content/spin/serverless-ai-tutorial.md

```diff
@@ -106,24 +106,79 @@ HTTP path: /api/...
 {{ blockEnd }}
 {{ blockEnd }}
 
-### Fetch AI Model
+### Fetching Supported AI Models
 
-Next, we create a folder and fetch a pre-trained AI model for our application:
+Next, **from within the application directory (alongside our `spin.toml` file), we create the appropriate folder structure (depending on which model we want to use) and then fetch the pre-trained AI model for our application:
 
-> Please note: this step can take a few minutes.
+> Please note: downloading a model can take a few minutes.
+
+#### llama2-chat example download
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ cd sentiment-analysis
+# llama2-chat
 $ mkdir -p .spin/ai-models
+$ cd .spin/ai-models
 $ wget https://huggingface.co/TheBloke/Llama-2-13B-chat-GGML/resolve/main/llama-2-13b-chat.ggmlv3.q3_K_L.bin
```

Comment:
> we should stop pointing to main on HuggingFace for the model

@radu-matei - what's the alterantive?

Reply:
This has been updated now, to cover off the fact that users not signing up for Serverless AI private beta, need to agree to Meta’s policies.